addons/survey/static/src/js/survey_result.js

```diff
@@ -178,6 +164,10 @@ publicWidget.registry.SurveyResultWidget = publicWidget.Widget.extend({
      * @private
      */
     _onPrintResultsClick: function () {
+        // Uncollapse all paginators on the page before printing.
```

Comment:
Could you please specify here or in the commit message that it was chosen for simplicity not to restore the existing state afterwards?